pkg/system/phase2_creating.go

```diff
@@ -484,6 +484,14 @@ func (r *Reconciler) SetDesiredCoreApp() error {
 				}}
 				util.MergeVolumeMountList(&c.VolumeMounts, &secretVolumeMounts)
 			}
+			if r.NooBaa.Spec.BucketLogging.LoggingType == nbv1.BucketLoggingTypeGuaranteed &&
+				util.KubeCheck(r.BucketLoggingPVC) {
```

Comment:
Do we need this check? didn't we already verify it exists?

Reply:
We can remove it. Thanks